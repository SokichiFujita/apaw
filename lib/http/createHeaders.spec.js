const createHeaders = require('./createHeaders');

describe('createHeaders', () => {
  it('should create an object representing HTTP request headers', () => {
    const headers = createHeaders({
      host: 'webservices.amazon.com',
      longDate: '20200308T123600Z',
      operate: 'SearchItems',
    });
    expect(headers).toMatchSnapshot();
  });
});
