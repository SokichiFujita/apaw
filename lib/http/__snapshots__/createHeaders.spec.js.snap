// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createHeaders should create an object representing HTTP request headers 1`] = `
Object {
  "content-encoding": "amz-1.0",
  "content-type": "application/json; charset=utf-8",
  "host": "webservices.amazon.com",
  "x-amz-date": "20200308T123600Z",
  "x-amz-target": "com.amazon.paapi5.v1.ProductAdvertisingAPIv1.undefined",
}
`;
